core/new-gui/src/app/dashboard/component/user-avatar/user-avatar.component.scss

```diff
@@ -0,0 +1,19 @@
+.texera-common-user-avatar {
+  object-fit: cover;
+  border-radius: 50%;
+  width: 38px;
+  height: 38px;
+  vertical-align: top;
+}
+
+.texera-normal-user-avatar {
+  font-size: smaller;
+  font-family: roboto, arial;
+  color: white;
+  line-height: 38px;
+  text-align: center;
+}
+
+.texera-common-user-avatar:hover {
+  box-shadow: #ccc 0px 0px 10px;
+}
```

Comment:
I see you defined two classes with two different styles. what's the purpose for each class, and when to use which? please leave some documentation for future reference.

Reply:
Has combined these classes into one. No need to separate because google user avatar a image, styles on text are for normal user.